onnxruntime/test/providers/cpu/tensor/quantize_linear_test.cc

```diff
@@ -55,6 +55,16 @@ TEST(DequantizeLinearOpTest, DequantizeLinear_Scalar) {
   test.Run();
 }
 
+// dequantize without zero point
+TEST( DequantizeLinearOpTest, DequantizeLinear_Without_Zero_Point )
```

Comment:
The spacing used in this test is different than the surrounding code and should be consistent.

Reply:
forgot to run clang-format.